src/shared/configUtils.ts

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Consolidating `identityUtils.ts` and `lwcDevServerUtils.ts` into `configUtils.ts` since the code in both of those files were around reading/writing config values.